CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 
 ### Added
 
+- [#5945](https://github.com/thanos-io/thanos/pull/5945) Downsample: Add flag to mark blocks for `no-downsample`, similar to the `no-compact.json` marker.
```

Comment:
Let's make this clearer. We added the new `no-downsample` marker to skip blocks when downsampling. And we added a new flag to mark blocks in the bucket tools rather than the downsample component.

Reply:
Can we separate them? We did 2 things so please mention both the tools change and the compact change